c10/core/TensorImpl.h

```diff
@@ -1526,6 +1545,7 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
   // should pack this into a bitfield.
   TensorTypeId type_id_;
   bool is_contiguous_ = true;
+  bool is_channels_last_contiguous_ = false;
```

Comment:
What's going on here? You don't ever actually read from this boolean in this PR (as far as I can tell). Is this the tag? Is this boolean advisory or not?

Reply:
At the very least, there should be a comment above this member...

Reply:
Looks like changes from my next PR leaked into this one. I will remove this lines.